README.md

```diff
@@ -73,7 +73,13 @@ Force fields moving forward will be called `name-X.Y.Z`
 
 ## Versions
 
-`v1.1.0 Parsley `: This minor release contains following changes. (1) Addition of new proper torsions and improper torsions for tetrazole; (2) Corrections to N-N bond rotation periodicity; (3) Removal of redundant periodicity component in `t19`; (4) Addition of three new bond and angle terms, `a22a`, `b14a` and `b36a`.
+- `v1.0.0 Parsley` : First major forcevield release.
```

Comment:
Typo, "forcefield" not "forcevield" ;)

Reply:
Ah, thanks for the catch! I'll fix this in the new, separate PRs.